docs/gcp/sql_proxy.md

```diff
@@ -0,0 +1,39 @@
+## CloudSQL Proxy 🚧
+
+Previously in our SQL article we saw how to implement a CloudSQL Proxy in our pods. Now it's time to see how and how does it work.
+
+CloudSQL proxy should offer much more advantages than the private IP. Here are Google's arguments regarding this subject
+
+> Secure connections: The proxy automatically encrypts traffic to and from the database using TLS 1.2 with a 128-bit AES cipher; SSL certificates are used to verify client and server identities.
```

Comment:
Source for quote : https://cloud.google.com/sql/docs/mysql/sql-proxy?hl=en#what_the_proxy_provides
Maybe add ¹ and put it at the bottom of the article ?

Reply:
Main source is at the bottom